packages/netlify-cms-locales/src/nl/index.js

```diff
@@ -83,10 +83,12 @@ const nl = {
       publishing: 'Publiceren...',
       publish: 'Publiceer',
       published: 'Gepubliceerd',
-      unpublished: 'Publicatie ongedaan maken',
+      unpublish: 'Publicatie terugtrekken',
+      unpublished: 'Publicatie teruggetrokken',
       duplicate: 'Dupliceren',
       unpublishing: 'Publicatie ongedaan maken...',
       publishAndCreateNew: 'Publiceer en maak nieuw item aan',
+      publishAndDuplicate: 'Publiceer en dupliceer',
```

Comment:
In het kader van consistentie een kleine wijziging: Publiceer en dupliceer item. 

Bedankt!